lib/MetricsConsumer.js

```diff
@@ -58,35 +68,68 @@ class MetricsConsumer {
         });
     }
 
+    _getRedisKeys(extension) {
```

Comment:
Not sure how relevant it is or how intrusive relative to existing deployments (backward compat), but we may be able to just use the extension name to format the redis keys, instead of using a static lookup. That would make the consumer more generic and agnostic of which extension is using it. But I understand it might not be practically doable. Another option though can be to publish the redis key format in the extension manifest (e.g. extensions/replication/index.js).

Reply:
ah ya I really like the idea of formatting the redis keys by the entries' `extension` received. 

There were no options for users to change the format of redis keys, so I think if we keep the same format, there should be no backwards compat issues. The replication metrics entry extension `crr` matches with the redis key as well. Some tweaking for test keys would be needed, but that should be fine